### PR TITLE
Ensure travis commands run as 'CI' environment.

### PR DESCRIPTION
**Pull request by dpagini**
Testing via patch file.

_Sorry, meant to test this against my fork. Will reopen if I think it's needed, still testing._

### CODE REVIEW

File: scripts/travis/run_tests [hunk 1]
```diff
@@ -6,7 +6,7 @@ set -ev
 yaml-cli update:value blt/project.yml project.local.hostname '127.0.0.1:8888'
 
 blt validate:all
-blt setup --define drush.alias='${drush.aliases.ci}' --no-interaction
+blt setup --define drush.alias='${drush.aliases.ci}' --define enviornment=ci --no-interaction
```
Comment (grasmash) [inline scripts/travis/run_tests]: Typo in "enviornment"

**Comment by grasmash** [comment]
But yes, good idea.

**Comment by dpagini** [comment]
K, cool... I'll re-open then. Thanks! Should I do the same for the pipelines file at all (not as familiar with that one)?

**Comment by grasmash** [comment]
Yes!

**Comment by grasmash** [comment]
Added to Pipelines in 4db3020f9

**Pull request merged**